### DISCUSSION TITLE
Audio track turned off on some files

### DISCUSSION THREAD

**Discussion by orbited303** [iina/iina#3845] [Q&A]
Dear IINA developers,

After updating to the new version the other week, I'm seeing some unwanted behavior: For some reason some videos have no sound. Quite a lot of them, actually.

Turns out that I need to manually select "Audio -> Audio Track -> #1" and the sound returns. This was never the case before the  update. Closing the file and opening it again, it retains the proper setting. 
 
Without being technologically adept enough to test my theory, I wonder if it could have something to do with these files not explicitly specifying a default track, and that there's been a change in how IINA treat such files? Where it previously just played the first track it can find IINA now defaults to not playing anything at all?

Anyone else who can reproduce this or experience the same thing?

**Comment by low-batt** [comment]
IINA 1.3.0 includes major updates from other projects to libraries used for audio and video playback. It is certainly possible that something has changed that the IINA project is not aware of.

This audio behavior should be entered as an IINA issue. Please attach `iina.log` and `mpv.log` log files when entering the issue. IINA logging can be enabled in the advanced section of IINA preferences.

We may need a video file that reproduces the problem. The usual way is to provide a small sample that reproduces the problem. When creating a small sample it is important that whatever program is used it does not re-encode the video. I believe this is the proper way to clip 15 seconds from the start of a video without re-encoding using [FFmpeg](https://ffmpeg.org/):
```text
ffmpeg -i long-input.mp4 -to 00:00:15 -c:v copy -c:a copy short-output.mp4
```
